features/git-prune-branches/feature_branch/behind_main/with_open_changes.feature

```diff
@@ -6,27 +6,49 @@ Feature: git prune-branches: don't remove the current empty feature branch if th
 
 
   Background:
-    Given I have a feature branch named "feature" behind main
-    And I have a feature branch named "stale_feature" behind main
-    And I am on the "feature" branch
+    Given the following commits exist in my repository
+      | BRANCH | LOCATION         | MESSAGE     | FILE NAME |
+      | main   | local and remote | main commit | main_file |
+    And I have a feature branch cut from "Initial commit" named "stale_feature_initial"
+    And I have a feature branch cut from "main commit" named "stale_feature_main"
```

Comment:
Instead of 
```
And I have a feature branch cut from "Initial commit" named "stale_feature_initial"
```
I would prefer
```
And I have a feature branch named "stale_feature" at "Initial commit"
```